File: 백준/Bronze/14681. 사분면 고르기/사분면 고르기.js
const fs = require('fs');

const readLine = fs.readFileSync(0, 'utf8').trim();

const [x, y] = readLine.split('\n').map(Number);
let quadrant = 0;

if (x > 0 && y > 0) {
    quadrant = 1;
} else if (x < 0 && y > 0) {
    quadrant = 2;
} else if (x < 0 && y < 0) {
    quadrant = 3;
} else {
    quadrant = 4;
}

console.log(quadrant);

File: 백준/Bronze/14681. 사분면 고르기/사분면 고르기.py
import sys

readLine = sys.stdin.readline

x = int(readLine())
y = int(readLine())
quadrant = 0

if x > 0 and y > 0 :
    quadrant = 1
elif x < 0 and y > 0 :
    quadrant = 2
elif x < 0 and y < 0 :
    quadrant = 3
else :
    quadrant = 4

print(quadrant)